contracts/registry/Registry.sol

```diff
@@ -72,6 +68,9 @@ contract Registry is MelonCouncilOwnable {
     mapping (address => bool) public fundFactoryIsRegistered;
 
     // Funds
+    address immutable public MLN_TOKEN;
```

Comment:
Let's move both of these above the storage vars, under a `// Constants` comment (L54, before `// Assets`).